docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -1,46 +1,65 @@
 
-**Note:** The Mac executable may need an exception for an unidentified developer: [Open a Mac app from an unidentified developer](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac)
+## Running Zowe commands in daemon mode
 
-## Running zowe commands with the zowe executable
+When you run Zowe in daemon mode, you run all Zowe commands as you normally run them. The first time you run a command, it starts the daemon in the background automatically and runs your desired Zowe command. Since the first Zowe command starts the daemon, the first command usually runs slower than a traditional Zowe command. However, subsequent Zowe commands run significantly faster. The daemon continues to run in the background until you close your terminal window.
 
-Verify that you have installed [the @next release of Zowe CLI](cli-install-cli-next.md) and the [zowe native executable](#install-the-zowex-native-executable).
+If you want to run Zowe in daemon mode constantly, add the path of the $ZOWE_CLI_HOME/bin (.zowe/bin) executables to the directory path of your computer's environment variables. When you configure Zowe to constantly run in daemon mode, daemon mode starts automatically after you issue the first Zowe command. When you open multiple terminal windows and issue Zowe commands, the Zowe commands in all of the terminal windows run in daemon mode.
 
-Run any zowe command as you normally would. The first time you run any zowe command, the command will automatically start a daemon in the background. It will then run your desired command. Since that first command must start the daemon, that first zowe command will actually run slower than a traditional zowe command. However, every zowe command afterward will run significantly faster. The daemon will continue to run in the background until you close your terminal window.
+**Enable daemon mode**
 
-Example:
+The following steps describe how to enable daemon mode and how to (optionally) configure Zowe to run in daemon mode constantly.
+
+1. Open a terminal window and issue the following command:
+   ```
+   zowe daemon enable
+   ```
+   The command copies the Zowe executables for your operating system into the $ZOWE_CLI_HOME/bin (.zowe/bin) directory. The next command that you issue starts the daemon.
+2. (Optional) Add the path to the Zowe executables to the environment variables for your computer. For example:
+   ```
+   C:\Users\<user_ID>\.zowe\bin
+   ```
+   **Important!** Ensure that you position the path to your Zowe executables before the path into which NPM installed the Node.js script. For information about configuring environment variables, see the documentation for your computer's operating system.
 
-```text
-zowe --version
-Starting a background process to increase performance ...
-7.0.0-next.202110211759
+   **Note:** Complete the environment variable configuration step (Step 2) only one time.
 
-zowe --version
-7.0.0-next.202110211759
+Example:
```

Comment:
This example comes right after a description of enabling the daemon. It led me to believe that it was an example of enabling the daemon. I suggest changing the wording to something like this:
```
Example:
```
to 
```
Example of running Zowe commands with the daemon enabled:
```

It's your call.